assets/src/edit-story/components/canvas/test/_utils.js

```diff
@@ -111,6 +118,13 @@ export function TestDisplayElement({
       ...(inputConfigContext && inputConfigContext.allowedMimeTypes),
     },
   };
+  const layoutContext = {
+    state: {
+      pageWidth: 412,
+      pageHeight: 618,
```

Comment:
Maybe move to a constant?

Reply:
Already are, but now they're imported in 270b05da9.